compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -827,16 +831,6 @@ else ()
         list(APPEND BUILTIN_CFLAGS_${arch} -fforce-enable-int128)
       endif()
 
-      if(arch STREQUAL "aarch64")
-        add_custom_target(
-          lse_builtin_symlinks
-          BYPRODUCTS ${lse_builtins}
-          ${arm64_lse_commands}
-        )
-
-        set(deps_aarch64 lse_builtin_symlinks)
```

Comment:
What's the replacement for this bit? I.e. how are the symlinks being depended on now so that they get created?

Reply:
The symlinks are used as sources for the `clang_rt.builtins` target which makes them a dependency.